apps/explorer/lib/explorer/chain/cache/min_missing_block.ex

```diff
@@ -25,11 +27,12 @@ defmodule Explorer.Chain.Cache.MinMissingBlockNumber do
   end
 
   def fetch_min_missing_block do
-    result = Chain.fetch_min_missing_block_cache()
+    last_fetched_counter = Chain.get_last_fetched_counter(@counter_type)
+    result = Chain.fetch_min_missing_block_cache(Decimal.to_integer(last_fetched_counter))
```

Comment:
very minor: can we merge the integer cast into the above pipeline?

```elixir
last_fetched_counter = Chain.get_last_fetched_counter(@counter_type)
|> Decimal.to_integer()
```

it's not so idiomatic to nest function calls like this